<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carrega Camadas</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QPushButton" name="load_btn">
     <property name="text">
      <string>&amp;Carregar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="workspace_label">
      <property name="text">
       <string>√Årea de Trabalho :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="workspace_options">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Estilo :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="styles_options">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="load_menu">
     <property name="text">
      <string>Carregar menu</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QProgressBar" name="progress_load">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="db_label">
      <property name="text">
       <string>Banco de Dados :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="db_options">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Camadas</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QSplitter" name="splitter_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="layers_list">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="send_all_btn_1">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="send_selected_btn_1">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="call_all_btn_1">
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="call_selected_btn_1">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QListWidget" name="layers_list_input">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QSplitter" name="splitter_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLineEdit" name="search_all_layers">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;center&gt;&amp;lt;&amp;lt; Pesquisar &amp;gt;&amp;gt;&lt;/center
&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="search_selected_layers">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>471</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="only_geometry">
         <property name="text">
          <string>Apenas com geometria</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Regras</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="rules_list">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="send_all_btn_2">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="send_selected_btn_2">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="call_all_btn_2">
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="call_selected_btn_2">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QListWidget" name="rules_list_input">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSplitter" name="splitter_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLineEdit" name="search_all_rules">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;center&gt;&amp;lt;&amp;lt; Pesquisar &amp;gt;&amp;gt;&lt;/center
&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="search_selected_rules">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Insumos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QSplitter" name="splitter_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="files_list">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="send_all_btn_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="send_selected_btn_3">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="call_all_btn_3">
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="call_selected_btn_3">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QListWidget" name="files_list_input">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QSplitter" name="splitter_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLineEdit" name="search_all_files">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;center&gt;&amp;lt;&amp;lt; Pesquisar &amp;gt;&amp;gt;&lt;/center
&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="search_selected_files">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
